src/service/cli/generate.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require(`fs`);
+const path = require(`path`);
+
+const {
+  CATEGORIES,
+  SENTENCES,
+  TITLES,
+  FILE_NAME,
+  DEFAULT_COUNT,
+  DATES
+} = require(`../../constants`);
+
+const {
+  getRandomInt,
+  shuffle
+} = require(`../../utils`);
+
+const generateArticles = (count) => (
+  Array(count).fill({}).map(() => ({
```

Comment:
```suggestion
  new Array(count).fill({}).map(() => ({
```
конструкторы должны вызываться с new